src/Constants.ts

```diff
@@ -3,7 +3,10 @@ dotenv.config();
 
 import { BigNumber } from 'ethers';
 
-export const BUNDLER_URL = 'https://dev.balloondogs.network:4338';
+export const BUNDLER_URL_LIVE = 'https://dev.balloondogs.network:4338';
```

Comment:
why not having this as a map and just need to validate the env is exist in the map